performance/index.ts

```diff
@@ -15,6 +15,12 @@ async function main() {
   commandArguments.forEach((commandArgument: string) => {
     const [arg, value] = commandArgument.split("=");
     switch (arg) {
+      case "pending":
+        testParams.pending = parseInt(value);
+        break;
+      case "throughput":
+        testParams.throughput = parseInt(value);
+        break;
```

Comment:
not fully understand those metrics :)

Reply:
pending - how many txs should be available all the time in mempool for node to collect 
throughput - expected number of processed txs in each block (on avarage)